src/common/interflop.h

```diff
@@ -21,6 +31,37 @@ enum FCMP_PREDICATE {
   FCMP_TRUE,
 };
 
+typedef struct interflop_function_info
+{
+  // Indicate if the function is from library
+  char id[500];
+  // Indicate if the function is from library
+  char isLibraryFunction;
+  // Indicate if the function is intrinsic
+  char isIntrinsicFunction;
+  // Indicate if the function use float
+  char haveFloat;
+  // Indicate if the function use float
+  char haveDouble;
+  // mantissa precision for 64 bit float
+  int binary64_precision;
```

Comment:
All the remaining attributes do not belong here. They have nothing to do with generic function info and should not be in interflop.h. Reason: separation of responsabilities and encapsulation.